### ISSUE TITLE
Question: how does this action handle multi-line release notes?

### ISSUE THREAD

**Issue by tonyarnold** [wzieba/AppCenter-Github-Action#28]
I've been calling the AppCenter CLI tool directly, and attempting to use the contents of `${{github.event.release.body}}` to populate my release notes, however it appears that I need to do some pre-processing to convert the newlines/special characters to something more appropriate for use on the command line. 

It doesn't look like this action does anything to help here, but I was thinking maybe it should? 

Something simple to pre-process the incoming release notes text, or wrap the argument contents in a multiline argument directly would be super helpful.

**Comment by wzieba** [comment]
> It doesn't look like this action does anything to help here, but I was thinking maybe it should?

That's true, I didn't pay attention to multi-line release notes. For sure it's good idea, I will check the possibilities. Thanks for raising this issue!

**Comment by VladSumtsov** [comment]
Hi! Do you have any updates on this? :)

**Comment by wzieba** [comment]
hi @VladSumtsov 👋 no, sorry I didn't prioritize it - I don't have any updates on this. I'll happily welcome any PR but for now, I don't have enough time to do this myself.

**Comment by sgurdag-wd** [comment]
Here is how i do it : 

 ```
  releaseNotes: |+

                        Initiated by **${{github.actor}}** --- **Github Actions**
                        - ${{github.workflow}}
                        - ${{github.ref}}
                        - Build (${{github.run_number}})
```

The magic is starting with `|+` . Then each line in yaml file will be shown in new line in AppCenter.

**Comment by sgurdag** [comment]
Here is how i do it :

 releaseNotes: |+

                       Initiated by **${{github.actor}}** --- **Github Actions**
                       - ${{github.workflow}}
                       - ${{github.ref}}
                       - Build (${{github.run_number}})
The magic is starting with |+ . Then each line in yaml file will be shown in new line in AppCenter.

**Comment by tonyarnold** [comment]
@sgurdag that's an interesting bit of info - does it work if you push through a multi-line string as per this issue?

**Comment by sgurdag** [comment]
> @sgurdag that's an interesting bit of info - does it work if you push through a multi-line string as per this issue?

Hey @tonyarnold, Yes,It does. You can see my release note on appcenter here:

 <img width="495" alt="Screenshot 2021-06-24 at 20 46 41" src="https://user-images.githubusercontent.com/5859851/123323522-8534b280-d52d-11eb-903c-ee5e9cf1046d.png">

 It is the way of having multi-lines in YAML as it is explicitly mentioned in syntax doc here : https://docs.ansible.com/ansible/latest/reference_appendices/YAMLSyntax.html

**Comment by tonyarnold** [comment]
I'm dubious about whether this will work with a multi-line replacement like `${{github.event.release.body}}` — I'll have to give it a try for my next beta release.

**Comment by p-fischer** [comment]
@tonyarnold I tested this with a multi-line replacement. See this snippet:
```
releaseNotes: |
  ${{ steps.extract-release-notes.outputs.release_notes }}
```
Unfortunately, it didn't work.

Another good and clean solution to support multi-line release notes would be to provide the release notes in a file and to support the CLI parameter [release-notes-file](https://github.com/microsoft/appcenter-cli/blob/de4ea0a2ad0f65c7d06b546855b56b9084b05048/src/commands/distribute/groups/publish.ts#L44)

**Comment by p-fischer** [comment]
Update: The approach can work. It failed in my test because the release notes started with `- `.
However, if they don't start with a minus it does work.

I worked around the limitation by adding a title in the YAML-config:
```
releaseNotes: |+
  Release notes:
  ${{ steps.extract-release-notes.outputs.release_notes }}
```
